benchmark/infrastructure/BenchmarkResultToString.cpp

```diff
@@ -2,11 +2,12 @@
 // Chair of Algorithms and Data Structures.
 // Author: Andre Schlegel February of 2023, schlegea@informatik.uni-freiburg.de)
 
-#include "../benchmark/BenchmarkRecordToString.h"
+#include "../benchmark/infrastructure/BenchmarkResultToString.h"
 
+namespace ad_benchmark{
 // ___________________________________________________________________________
 void addCategoryTitelToOStringstream(std::ostringstream* stream,
-    const std::string categoryTitel){
+    std::string_view categoryTitel){
```

Comment:
It is `title`, not `titel` (maybe also search for this typo in the rest of the code).

Reply:
Done, see [c5b691a](https://github.com/ad-freiburg/qlever/pull/860/commits/c5b691a353268a8724a86b73246d10db55ccc00e).